lib/src/fields/form_builder_signature_pad.dart

```diff
@@ -135,35 +140,50 @@ class _FormBuilderSignaturePadState extends State<FormBuilderSignaturePad> {
                 decoration: BoxDecoration(
                   border: Border.all(color: Colors.grey),
                 ),
-                child: GestureDetector(
-                  onVerticalDragUpdate: (_) {},
-                  child: Signature(
-                    width: widget.width,
-                    height: widget.height,
-                    backgroundColor: widget.backgroundColor,
-                    controller: _effectiveController,
-                  ),
-                ),
+                child: _savedValue != null || _readOnly
+                    ? Card(
+                        color: widget.backgroundColor,
+                        child: Image.memory(
+                          _savedValue,
+                          height: widget.height,
+                          width: widget.width,
+                        ),
+                      )
+                    : GestureDetector(
+                        onVerticalDragUpdate: (_) {},
```

Comment:
This seems odd to me; a comment explaining why this is necessary would be wise.

Reply:
Added comments as requested

Reply:
My comment was questioning the existence of the `GestureDetector` with the `onVerticalDragUpdate` that basically does nothing.  The whole widget _looks_ like it is adding no value, but I suspect that it is doing something valuable.  (If not, we should get rid of it.)

Sorry @vin-fandemand, I see that it existed _before_ your change; this isn't a showstopper, just an opportunity to make it better.  Maybe @danvick can comment on it?

Again, not a showstopper, just something that I noticed when reviewing.

Reply:
@awhitford no problem